stories/notification/notification.stories.js

```diff
@@ -5,512 +5,295 @@ import '../../dist/notification.css';
-        <i class="sap-icon--decline"></i>
-    </button>
-</div>
-<div class="fd-notification__body">
-    <div class="fd-notification__content">
-        <div class="fd-notification__text">
-            <div class="fd-notification__description">
-                Default Notification Description <br>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-            </div>
-            <div class="fd-notification__metadata">
-                Default Notification Metadata
+    <div class="fd-notification__body">
+        <span class="fd-avatar fd-avatar--xs fd-avatar--circle fd-avatar--thumbnail" style="background-image: url('http://lorempixel.com/400/400/nature/4/')" role="presentation" aria-label="John Doe"></span>
+        <div class="fd-notification__content">
+            <div class="fd-notification__header">
+                <div class="fd-notification__indicator fd-notification__indicator--success"></div>
+                <h2>Your leave request has been accepted</h2>
             </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            <p>SAP Analytics Cloud &middot; 7 minutes ago</p>
         </div>
-    </div>
-    <div class="fd-notification__footer">
-        <button role="button" class="fd-button fd-button--transparent">More Info</button>
         <div class="fd-notification__actions">
-            <button role="button" class="fd-button fd-button--positive">Approve</button>
-            <button role="button" class="fd-button fd-button--negative">Reject</button>
+            <button class="fd-button fd-button--compact">Open</button>
+            <button class="fd-button fd-button--transparent fd-button--compact" aria-label="Close">
+                <i class="sap-icon--decline"></i>
+            </button>
         </div>
     </div>
 </div>
-</div>
 `;
 primary.parameters = {
     docs: {
         iframeHeight: 200
     }
 };
 
-export const success = () => `
-<div class="fd-notification fd-notification--success">
-    <div class="fd-notification__header">
-        <div class="fd-notification__indicator--success"></div>
-        <div class="fd-notification__title">This is the notifications title Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
-        <button aria-label="decline-button" role="button" class="fd-button fd-button--transparent fd-notification__close">
-            <i class="sap-icon--decline"></i>
-        </button>
-    </div>
+export const noAvatar = () => `
```

Comment:
So this got bolded accidentally because you need to put backslashes between the underscores like this "fd-notification\ _ \ _" (no spaces). Also they need to be wrapped in code.